main.c

```diff
@@ -246,6 +323,22 @@ int main(int argc, char *argv[], char *envp[])
 				opts.arch = optarg;
 				break;
 
+			case OPTION_LIMIT_CORE:
+				if (parse_rlimit(&opts.limit_core, optarg)) {
+					errx(1, "invalid / out of range --limit-core value");
```

Comment:
Maybe parse_rlimit and parse_complete_rlim_t could set the correct errno between EINVAL and EOVERFLOW so that we can use err() and get which error happened?